### ISSUE TITLE
Change help text for checkboxes

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab#122]
@mbarlow12, I updated the help text for the checkboxes in the design document based on some feedback.

**For reference, this is the current help text in the A11y View**
<img width="408" alt="checkboxes-a11y-view" src="https://user-images.githubusercontent.com/1902116/49726580-a3d44600-fc6e-11e8-9eff-6dc29591e003.png">

**Please change the help text of each checkbox to:**
- For Constant Size, checkbox
  - Keep size of spheres constant while changing mass.

- For Force Values, checkbox
  - Explore force values in micronewtons.

- For Scientific Notation, checkbox
  - Explore force values in scientific notation.

Here's a link to the [Control Area section](https://docs.google.com/document/d/1-37qAgde2XrlXBQae2SgjartM35_EnzDD9pdtd3nXAM/edit#heading=h.uyv8gvko1gkk) in design document, if needed

**Comment by mbarlow12** [comment]
@terracoda The updates should be complete in the above commits. Assigning to you for review/closing.

**Comment by terracoda** [comment]
@mbarlow12, changes sound great! Thanks!
I will close this issue!
